recipe_batches/recipe_batches.py

```diff
@@ -2,13 +2,26 @@
 
 import math
 
+
 def recipe_batches(recipe, ingredients):
-  pass 
+    difference = []
+    for recipe_ingredient in recipe:
+        if recipe_ingredient in ingredients:
+            difference.append(
+                ingredients[recipe_ingredient] / recipe[recipe_ingredient])
+        else:
+            difference = [0]
+            break
+    if min(difference) < 1:
+        return 0
+    else:
+        return round(min(difference) - 0.4)
```

Comment:
Why are you subtracting 0.4 here?

Reply:
This works anyway, good job!

Reply:
> 
> 
> Why are you subtracting 0.4 here?

This was to round down from anything larger than 1.8. For some reason placing .5 there skewed all the affect data.